.github/workflows/publish.yml

```
@@ -47,13 +167,51 @@ jobs:
 
     - name: Package client
       run: dotnet run --project Content.Packaging client --no-wipe-release
 
     - name: Publish version
-      run: Tools/publish_multi_request.py --fork-id sunrise_station
+      # Sunrise edit start - передаём настраиваемый идентификатор сборки
+      run: Tools/publish_multi_request.py --fork-id "${PUBLISH_FORK_ID}"
+      # Sunrise edit end
       env:
         PUBLISH_TOKEN: ${{ secrets.PUBLISH_TOKEN }}
         GITHUB_REPOSITORY: ${{ vars.GITHUB_REPOSITORY }}
```

actionlint (1.7.12)

[error] configuration variable name "github_repository" must not start with the GITHUB_ prefix (case insensitive). note: see the convention at https://docs.github.com/en/actions/learn-github-actions/variables#naming-conventions-for-configuration-variables

(expression)